pom.xml

```diff
@@ -284,7 +284,7 @@
   </dependencies>
 
   <build>
-    <testSourceDirectory>src/test/java</testSourceDirectory>
+    <testSourceDirectory>${testSourceLocation}</testSourceDirectory>
```

Comment:
Add `<testSourceLocation>${project.basedir}/src/test/java/</testSourceLocation>` in properties starting from line 76

Reply:
Added the Suggested change